tools/metadata/meta/schema.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from typing import Optional, List, Any
+
+from ..schema import SchemaValue, validate_dict
+
+"""
+YAML filename for meta files
+"""
+META_YML_FILENAME = "META.yml"
+
+@dataclass
+class MetaFile():
+    """documented structure of META files.
+    Reference: https://github.com/web-platform-tests/wpt/pull/18434
```

Comment:
Hmm, it's unfortunate the documentation of `META.yml` is spread across two files. But I agree this PR is then the best thing to point to.